src/Orc.NuGetExplorer/Orc.NuGetExplorer.Shared/Models/PackageDetails.cs

```diff
@@ -37,8 +50,16 @@ internal PackageDetails(IPackage package)
         #endregion
 
         #region Properties
-        public List<string> ApiValidations { get; set; } = new List<string>();
+        public DateTime LastModified => _validationContext.LastModified;
```

Comment:
This is wrong. It exposes "LastModified", which reflects to the package, but then it returns the validation context. Don't expose it like this. Why not just expose IValidationContext on the package?